app/templates/application.hbs

```diff
@@ -14,6 +14,6 @@
 <body>
   {{outlet}}
 </body>
-{{#if (not-eq session.data.authenticated.profile.name 'MaryJane ')}}
+{{#if (not-eq session.data.authenticated.profile.name 'Allison Reu Singer')}}
```

Comment:
Might need to switch this to one of us to be able to view the admin-dashboard while in production.